rskj-core/src/test/java/co/rsk/peg/BridgeSupportTest.java

```diff
@@ -6229,6 +6228,292 @@ void receiveHeader_block_exist_in_storage() throws IOException, BlockStoreExcept
         Assertions.assertEquals(-4, result);
     }
 
+    @Nested
+    @TestInstance(TestInstance.Lifecycle.PER_CLASS)
+    @Tag("test chain work before and after rskip 434")
+    class ChainWorkTests {
+        ActivationConfig.ForBlock activations;
+        BridgeConstants bridgeMainnetConstants = BridgeMainNetConstants.getInstance();
+        Repository repository;
+        BridgeStorageProvider bridgeStorageProvider;
+        BtcBlockStoreWithCache.Factory btcBlockStoreFactory;
+        BtcBlockStoreWithCache btcBlockStoreWithCache;
+        BridgeSupportBuilder bridgeSupportBuilder = new BridgeSupportBuilder();
+        BridgeSupport bridgeSupport;
+
+        BtcBlock block849134;
+        BtcBlock block849135;
+        BtcBlock block849136;
+        BtcBlock block849137;
+        BtcBlock blockWithTooMuchWork;
+        BtcBlock block849139;
```

Comment:
```suggestion
        BtcBlock block849_134;
        BtcBlock block849_135;
        BtcBlock block849_136;
        BtcBlock block849_137;
        BtcBlock blockWithTooMuchWork;
        BtcBlock block849_139;
```

To me, this can be a constant and can be initialized right here. So, you can name it uppercase.

Reply:
i find better the way it is now